executor/aggfuncs/func_sum.go

```diff
@@ -155,6 +189,40 @@ type sum4Decimal struct {
 	baseSumAggFunc
 }
 
+func (c *sum4Decimal) SerializeForSpill(_ sessionctx.Context, partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper) {
+	pr := (*partialResult4SumFloat64)(partialResult)
+	resBuf := spillHelper.serializePartialResult4SumFloat64(*pr)
```

Comment:
serializePartialResult4SumFloat64

this should be decimal

Reply:
> 



> serializePartialResult4SumFloat64
> 
> this should be decimal

done